src/qmake/platforms/linux.pri

```diff
@@ -136,20 +139,33 @@ systemd_service.files = $$PWD/../../../linux/mozillavpn.service
 systemd_service.path = $${USRPATH}/lib/systemd/system
 INSTALLS += systemd_service
 
-manifestFirefox.files = $$PWD/../../../extension/manifests/linux/mozillavpn.json
-manifestFirefox.path = $${USRPATH}/lib/mozilla/native-messaging-hosts
+ORIG_MOZILLAVPN_JSON = $$PWD/../../../extension/manifests/linux/mozillavpn.json
+manifestFile.input = ORIG_MOZILLAVPN_JSON
+manifestFile.output = $$PWD/../../../linux/mozillavpn.json
+manifestFile.commands = @$(SED) \'/\"path\":/c\ \ \"path\":\ \"$${LIBPATH}/mozillavpn/mozillavpnnp\",\' $$ORIG_MOZILLAVPN_JSON >  $$manifestFile.output
```

Comment:
Does it work on Windows too? Is there anything we can do using a non posix command? Maybe a python script...

Reply:
I think the `linux.pri` is only ever called on on linux no? I can definitely try a python hack instead if you want to support some non-posix-to-posix cross compiling or something of the sort.